apps/web/src/app/(unauthenticated)/check-email/page.tsx

```diff
@@ -1,7 +1,12 @@
+import type { Metadata } from 'next';
 import Link from 'next/link';
 
 import { Button } from '@documenso/ui/primitives/button';
 
+export const metadata: Metadata = {
+  title: 'Check Email',
+};
```

Comment:
I think it would be better to have this as `Forgot password` since it's part of the forgot password flow